<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModelSearchWidget</class>
 <widget class="QWidget" name="ModelSearchWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>模型搜索</string>
  </property>
  <layout class="QVBoxLayout" name="ly_total">
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="ly_search" stretch="1,1,3">
     <item>
      <widget class="QPushButton" name="bt_browser">
       <property name="text">
        <string>浏览</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bt_search">
       <property name="text">
        <string>搜索</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_input"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="ly_preview"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
